Dockerfile

```
@@ -4,7 +4,7 @@ FROM --platform=$BUILDPLATFORM node:22-alpine3.20 AS build-stage
 ENV PNPM_HOME="/pnpm"
 ENV PATH="$PNPM_HOME:$PATH"
 
-RUN corepack enable && corepack use pnpm@9
+RUN npm install -g pnpm@9
```

I think it could be great to follow the the pnpm official doc, which mention the key change issue, and suggest to update corepack (cf [doc](https://pnpm.io/installation#using-corepack)). 
```suggestion
RUN npm install --global corepack@latest
RUN corepack enable && corepack use pnpm@9
```

---

Just pushed a rebased branch w/the requested change.